client/src/components/GroupCard/GroupCard.css

```diff
@@ -0,0 +1,36 @@
+.GroupCard {
+    width: 17vw;
+    height: 19vh;
```

Comment:
`vw` and `vh` are great for big pieces of the UI (like headers). Could you use percent or rem here?

Reply:
I should have specified earlier 🙉

Reply:
@shelbyvjacobs Did you forget this one?